Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,60 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+## Name
+Tips and Tricks
+
+## Description
+This is a list of useful tips and tricks to help you make the most out of SerenityOS.
+
+## General
+* When on the Desktop or in File Manager, start typing and the nearest match will be selected.
+* Bold text in context menus hints at the default behavior of a double-click.
+* Hold `Ctrl` to accelerate mouse wheel interaction with sliders and spin boxes.
+* Hold `Ctrl` while activating a menu item to prevent that menu from closing.
+* Many applications will open a compatible file if you drag-and-drop it into their window.
+* Default file and protocol associations can be changed in `~/.config/LaunchServer.ini`.
+
+### Window Management
+* Double-click a window's title bar to maximize it.
+* Click on a window's icon to open the context menu.
+* Double-click on the edge of an application's window to maximize it in that direction.
+* Middle-click on a window's maximize button to extend the window vertically (this can be undone in the same way).
+* Resizable windows can be dragged to all sides or corners of the screen, where they will automatically resize to fill half or one-quarter of the screen.
+* Right-click on the Workspace Picker applet and select 'Workspace Settings' to easily customize the number and layout of Workspaces (virtual desktops).
+
+### Fun
+* It can help to get a second pair of `$ Eyes` on a problem… or fifty: `$ Eyes -n 100`.
+
+## Applications
+
+### [Assistant](help://man/1/Applications/Assistant)
+* Assistant can help you to quickly find files and applications. Summon it with `Super+Space`.
```

Comment:
not really sure with that tip… it feels detached from the second one. i believe all tips should be standalone.

the first tip says in fact what the assistant is and how to open it with a keystroke…

hard to tell here, really!

i think that the second tip is a real tip; maybe you could rephrase the tips, so you could tell:
* how to open the assistant (dubious?)
* that you can open applications (more dubious?)
* that you can find files and open them in the default application
* that you can open urls in the default application (e.g. Browser)
* that can do simple computations in it (i believe itʼs possible)

files and urls could possibly be merged as those are virtually the same in some way…? 🤔

Reply:
I'm trying to keep this short and sweet:

- The first tip quickly tells you there is a system search/launcher and how to access it.
- The second tips lists the only other functionality (that I know of).

I also thought you could do simple sums in Assistant, but have never managed it. Did we both dream about this functionality or was there a PR once upon a time (maybe it was never merged)?